### PR TITLE
Fixes for reusing SRPM logic and for showing correct latest sign status on build feed

### PR DESCRIPTION
**Pull request by anfimovdm**

### CODE REVIEW

File: alws/crud/build_node.py [hunk 1]
```diff
@@ -565,11 +556,23 @@ async def build_done(db: Session, pulp: PulpClient, request: build_node_schema.B
     db_srpm_name = None
     srpm = None
     for rpm in rpms_result.scalars().all():
+        # if build task is excluded or failed, we don't need to create
+        # SourceRpm and BinaryRpm records, because it's break package releases
+        if status in (BuildTaskStatus.EXCLUDED, BuildTaskStatus.FAILED):
```
Comment (Korulag) [inline alws/crud/build_node.py:561]: So you will not produce SourceRpm instance, but build artifact will be there, correct?

Comment (anfimovdm) [reply alws/crud/build_node.py:561]: That’s correct

File: alws/crud/build_node.py [hunk 2]
```diff
@@ -249,27 +249,17 @@ def get_repo(repo_arch, is_debug):
...
-                    models.SourceRpm.build_id == task_query).options(
-                    selectinload(models.SourceRpm.artifact)
-                )
-                res = await db.execute(srpm_query)
-                srpms = res.scalars().all()
-                srpm_href = next(
-                    i.artifact.href for i in srpms
-                    if i.artifact.name == srpm.name
-                )
+            db_srpm = await db.execute(
```
[the remark this answers is not in the document]
Comment (anfimovdm) [reply alws/crud/build_node.py:252]: Good point, I'll fix it

[the remark this answers is not in the document]
Comment (anfimovdm) [reply alws/crud/build_node.py]: Done

**Pull request merged**